spec/features/course/unread_status_management_spec.rb

```diff
@@ -52,8 +52,7 @@
 
         it 'shows the correct number of unread items' do
           expect(course.announcements.unread_by(first_user).count).to eq(1)
-          expect(page).to have_link(format('%s (%d)', I18n.t('course.announcements.sidebar_title'),
-                                           1))
+          expect(page).to have_link(I18n.t('course.announcements.sidebar_title'))
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.